app/controllers/api/v1/search_results_controller.rb

```diff
@@ -45,7 +45,9 @@ def create_search_results_query
         SearchResultsQuery.new(
           current_user.search_results, {
             url_equals: filter_params[:url_equals],
-            adwords_url_contains: filter_params[:adwords_url_contains]
+            adwords_url_contains: filter_params[:adwords_url_contains],
+            url_contains_at_least_one: filter_params[:url_contains_at_least_one],
+            url_contains_at_least_two: filter_params[:url_contains_at_least_two]
```

Comment:
These filters are not extensible 🙈 How about breaking them into two instead:

```
url_contains: filter_params[:url_contains],
match_count: filter_params[:match_count],
```

So we could search for 1, 2, ♾️ number of occurrences 💡